h/test/tweens_test.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.testing import DummyRequest as _DummyRequest
+from h import tweens
+
+
+class DummyRegistry():
+    def __init__(self, *args, **kwargs):
+        self.settings = {}
+
+
+class DummyRequest(_DummyRequest):
+    def __init__(self, *args, **kwargs):
+        self.registry = DummyRegistry()
+        super(DummyRequest, self).__init__(*args, **kwargs)
+
+
+class DummyHeader():
```

Comment:
DummyHeader? It looks like this is supposed to be a dummy handler? If so, I'd suggest you just use `handler = lambda request: None` in the tests.